Update UG (#197)
AY2021S1-CS2103T-F11-1/tp: docs/UserGuide.md

Shows a list of all students in charge by the TA.

> You may navigate to the Students tab to view the full list of students.

Reviewer: ```suggestion
You may click on the Students tab to view the full list of students.
```
Perhaps this phrasing will be less vague?

### Listing all consultations: `list-consult`

> Shows a list of all consultations in charge by the TA. 

Reviewer: ```suggestion
Shows a list of all consultations that has been arranged with your students. 
```
